lib/live_debugger_refactor/services/callback_tracer/events.ex

```diff
@@ -0,0 +1,16 @@
+defmodule LiveDebuggerRefactor.Services.CallbackTracer.Events do
+  @moduledoc """
+  Events emitted by CallbackTracer service.
+  """
+
+  use LiveDebuggerRefactor.Event
+
+  alias LiveDebugger.CommonTypes
+
+  defevent(
+    TraceReturned,
+    module: module(),
+    cid: CommonTypes.cid(),
+    function: atom()
+  )
```

Comment:
We include `PID` directly in event now.
```suggestion
  defevent(
    TraceReturned,
    module: module(),
    function: atom(),
    pid: pid(),
    cid: CommonTypes.cid()
  )
```

Reply:
I'll wait untill you merge your PR and then resovlve conflicts